src/iblt.cpp

```diff
@@ -117,9 +117,14 @@ void CIblt::_insert(int plusOrMinus, uint64_t k, const std::vector<uint8_t> v)
 {
     // assert(v.size() == valueSize);
 
+    if (!n_hash)
```

Comment:
Since n_hash is only set to a variable quantity in resize, why not throw an assertion error or raise an exception there if n_hash is 0? Really the IBLT won't work at all without even a single hash function.

Reply:
Yes, I am fine with either. I just want to make sure that the IBLT has no state where it will create crashing DIV-0 errors. When going the exception route, one should make sure that all IBLTs that are accessed after they have been deserialized are only handled in sections that are properly sitting in `try { ... } catch (std::exception) ..` blocks. I did not check whether that is the case yet(?)

This applies similarly to your comment below as well.